includes/utils.sh

```diff
@@ -256,15 +256,15 @@ then
 		if [ "${RET0}" != "" ]
 		then
 			RET2=`eval norminette $RET0 2>&1`
+			RET4=$?
 			echo "$RET2" > "$RETURNPATH"/.mynorminette
 			RET2=`cat .mynorminette | grep Error`
 			RET3=`cat .mynorminette | grep Warning`
-			RET4=`cat .mynorminette | grep "command not found"`
 			if [ "$RET2" == "" -a "$RET3" == "" -a "$RET4" == "" ]
```

Comment:
never be empty ? if last command succeed, it will be 0; else >0 nop ?

eh yes, you're probably right for eval instead of norminette; but on my pc `eval` fail if the command evaluated fail ( or at least, the last return value is set to the failed command )